tiledb/sm/cpp_api/query_condition.h

```diff
@@ -143,7 +143,7 @@ class QueryCondition {
    *
    * int cmp_value = 5;
```

Comment:
Looks like `cmp_value` isn't used here.

Reply:
Nice catch. That is an oversight when I corrected to use a string comparison.  Will make a a string and use it.